stubs/redis/redis/sentinel.pyi

```diff
@@ -37,10 +36,10 @@ class SentinelConnectionPool(ConnectionPool):
     def rotate_slaves(self) -> Iterator[_AddressAndPort]: ...
 
 class Sentinel(SentinelCommands):
-    sentinel_kwargs: Any
+    sentinel_kwargs: dict[str, Any | None]
```

Comment:
Why `Any | None` here, rather than just `Any`?

Reply:
To keep it in sync with `__init__`: https://github.com/python/typeshed/pull/8661/files#diff-e6904f5cf832e0aa2d8b88f35c960d621184111abe36685bc4c4f53cb8398c82R47

Reply:
I think we should change the annotation in `__init__` to `dict[str, Any] | None` rather than `Any | None`.

And, if `None` is passed into `__init__`, the `sentinel_kwargs` attribute is set to a different dictionary: https://github.com/redis/redis-py/blob/0b27e3ebdfd7e7dd67222bcb043fcdfcc907bd42/redis/sentinel.py#L179-L182. So the `sentinel_kwargs` attribute can never be `None`. So I think this should just be `dict[str, Any]`

```suggestion
    sentinel_kwargs: dict[str, Any]
```

Reply:
Good catch.